x-pack/filebeat/module/squid/log/test/access1.log-expected.json

```diff
@@ -0,0 +1,5610 @@
+[
+    {
+        "@timestamp": "2006-09-08T04:21:52.000Z",
+        "destination.as.number": 36752,
+        "destination.as.organization.name": "Oath Holdings Inc.",
+        "destination.geo.continent_name": "North America",
+        "destination.geo.country_iso_code": "US",
+        "destination.geo.location.lat": 37.751,
+        "destination.geo.location.lon": -97.822,
+        "destination.ip": [
+            "209.73.177.115"
+        ],
+        "event.action": "TCP_MISS",
+        "event.code": "CONNECT",
+        "event.dataset": "squid.log",
+        "event.module": "squid",
+        "event.original": "1157689312.049   5006 10.105.21.199 TCP_MISS/200 19763 CONNECT login.yahoo.com:443 badeyek DIRECT/209.73.177.115 -",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.offset": 0,
+        "observer.product": "Proxy",
+        "observer.type": "Proxies",
+        "observer.vendor": "Squid",
+        "related.ip": [
+            "10.105.21.199",
+            "209.73.177.115"
+        ],
+        "rsa.internal.hcode": "DIRECT",
+        "rsa.internal.messageid": "CONNECT",
+        "rsa.investigations.ec_subject": "NetworkComm",
+        "rsa.investigations.ec_theme": "ALM",
+        "rsa.misc.action": [
+            "CONNECT",
+            "TCP_MISS"
+        ],
+        "rsa.misc.content_type": "-",
+        "rsa.misc.result_code": "200",
+        "rsa.network.domain": "login.yahoo.com",
+        "rsa.time.duration_time": 5006,
+        "rsa.time.event_time": "2006-09-08T04:21:52.000Z",
+        "rsa.time.event_time_str": "1157689312",
+        "rsa.web.alias_host": "login.yahoo.com",
+        "server.domain": "login.yahoo.com",
```

Comment:
@andrewstucki Yes as you stated `client/server` is appropriate where there's a some type of session opened and each of the two systems have a clear role as either the `client` or `server` in the transaction. `source/destination` isn't limited to an unestablished or stateless connection; a `client/server` TCP session event could also populate `source/destination`.

In the case of a (assuming explicit) proxy, you would have two unique TCP connections established: one from host to proxy and another from proxy to origin server. Squid doesn't appear to log the events this way though (consolidating the two separate sessions into a single event in the logs). Since what is logged is the source host and the destination origin from the observer (squid), I think `source/destination` is appropriate in this case.